NorthstarDLL/scripts/scriptutility.cpp

```diff
@@ -26,3 +27,21 @@ ADD_SQFUNC(
 
 	return SQRESULT_NULL;
 }
+
+ADD_SQFUNC(
+	"string",
+	GetNorthstarVersion,
```

Comment:
Should we call the function `GetNorthstarLauncherVersion` or something alike to indicate that it's the version of the native code and not script? ^^"

Reply:
I considered that, but `GetNorthstarLauncherVersion` implies that it is getting the version of `NorthstarLauncher.exe` which isn't the case.

I don't think that this would cause issues, given that getting script version is more verbose `NSGetModVersion` or whatever it is exactly.

For most users, the versions should all match up anyway, this is mostly meant for dev builds and such. The overarching goal for me here is to display more build info in squirrel, so that the exact build of clients can be determined from something as simple as a screenshot of the error dialogue